plugins/zmq/src/plugin.cpp

```diff
@@ -552,6 +552,10 @@ class plugin final : public virtual loader_plugin<zmq_loader>,
   auto name() const -> std::string override {
     return "zmq";
   }
+
+  auto supported_uri_schemes() const -> std::vector<std::string> override {
+    return {"zmq", "inproc"};
```

Comment:
While I see the point, I'm not sure if `inproc` is a good *global* scheme. Alternatively we might add `zmq-tcp` and `zmq-inproc`, with `zmq` defaulting to the former.

I have a mild opinion on this, though.

Reply:
I prefer it this way. The inproc scheme is not used by anything else so far. Since you said that you only have a mild opinion I'll not change this now, and we can choose to revisit this later on.